rocketc/rocketc.py

```diff
@@ -109,29 +111,39 @@ def init(self):
         else:
             return response['errorType']
 
+    def get_url_api_rocket_chat(self):
+        """
+        This method retunrs the rocketChat url service where someone can acces to API
+        """
+        xblock_settings = self.get_xblock_settings()
+        if "url_service" in xblock_settings:
+            return "/".join([xblock_settings["url_service"], "api", "v1"])
+
+        return "/".join(["http://localhost:3000", "api", "v1"])
```

Comment:
will this eventually changed by a setting? the base rocket chat url?

Reply:
Sorry ignore this comment, this is just a fallback if the setting is not defined